origin-js/scripts/serve.js

```diff
@@ -17,4 +19,141 @@ const start = async () => {
   await startIpfs()
 }
 
+
+class StoreMock {
+  constructor() {
+    this.storage = {}
+  }
+
+  get(key) {
+    return this.storage[key]
+  }
+
+  set(key, value) {
+    this.storage[key] = value
+  }
+}
+
+const createSamples = async() => {
+  const provider = new Web3.providers.HttpProvider('http://localhost:8545')
+  const web3 = new Web3(provider)
+  const accounts = await web3.eth.getAccounts()
+  const contractService = await contractServiceHelper(web3)
+  const ipfsService = new IpfsService({
+    ipfsDomain: '127.0.0.1',
+    ipfsApiPort: '5002',
+    ipfsGatewayPort: '8080',
+    ipfsGatewayProtocol: 'http'
+  })
+  const store = new StoreMock()
+  store.set(
+    'notification_subscription_start',
+    new Date().getTime()
+  )
+  marketplace = new Marketplace({
```

Comment:
As opposed to instantiating directly a Marketplace object, how about creating an Origin object which is the base for using the origin-js APIs ?
See an example [here](https://github.com/OriginProtocol/origin/blob/master/origin-tests/test/1-create-listing.js#L19).
Also see our (still work in progress) documentation [here](https://docs.originprotocol.com/#origin-object).

In order to import Origin, could you try [this](https://github.com/OriginProtocol/origin/blob/master/origin-tests/test/1-create-listing.js#L8) and see if that works ?

Reply:
Yes. I will try that. I initially did use an Origin object but I ran into the same problem of imports.

Reply:
I changed the code to use and Origin object and also to use sample fixtures. However, that style of importing origin still doesn't work

Reply:
hmm... what error do you get exactly ?

Reply:
` /app/origin-js/src/index.js:1
origin                  | 
(function (exports, require, module, __filename, __dirname) { import ContractService from './services/contract-service'
origin                  |                                                                      ^^^^^^^^^^^^^^^
origin                  |
origin                  | SyntaxError: Unexpected identifier`

Reply:
You'll have to use `babel-node` to get support for ES6. E.g. `babel-node scripts/serve.js --preset @babel/env`

Reply:
hmm should I add babel-node to the dependencies?

Reply:
Sgtm. Please try adding babel-node and see if that helps...